plugin/keyring_vault/CMakeLists.txt

```diff
@@ -1,29 +1,27 @@
+IF (NOT DEFINED WITH_KEYRING_VAULT)
```

Comment:
Commit comment still mentions `FORCE` as a valid option for `WITH_KEYRING_VAULT`